domain/src/test/kotlin/no/nav/su/se/bakover/domain/vedtak/VedtakTest.kt

```diff
@@ -179,19 +193,35 @@ internal class VedtakTest {
                     periodeA,
                     bosituasjonA,
                 ),
+                // TODO jah: Ikke bra at dette ender opp som et innvilget vedtak
```

Comment:
Ny pr; stramme opp init på gyldige tilstander? F.eks sjekk at vilkår er innvilget, grunnlag konsistent etc. Kanskje til og med vurdere å skille på de fullstendige/ufullstendige typene en gang for alle?

Reply:
Se task: https://trello.com/c/F4au2IcL